components/ShareAchievementModal.vue

```diff
@@ -23,7 +23,30 @@
             alt="Avatar"
             class="rounded-full w-24 h-24 mx-auto mb-4"
           />
-          <h3 class="text-lg font-semibold pb-3">{{ userName }}</h3>
+          <!-- <h3 class="text-lg font-semibold pb-3">{{ userName }}</h3> -->
```

Comment:
commented code

Reply:
Resolved in c07c4dc3da0c337b795def225290ffd1c9bb4254